packages/kbn-grid-layout/grid/use_grid_layout_events.ts

```diff
@@ -195,7 +195,7 @@ export const useGridLayoutEvents = ({
       const atTheBottom = window.innerHeight + window.scrollY >= document.body.scrollHeight;
 
       if (!isTouchEvent(e)) {
-        const startScrollingUp = !isResize && heightPercentage < 5 && !atTheTop; // don't scroll up when resizing
+        const startScrollingUp = heightPercentage < 5 && !atTheTop; // don't scroll up when resizing
```

Comment:
Talked with @ThomThomson and we decided to allow scrolling up when resizing because it's useful when panels are taller than the viewport. You almost always end up with a panel that takes up only a single row when doing this though, which is why I disabled it originally 🙈 




https://github.com/user-attachments/assets/73338a96-81ea-4cdf-a91b-d8d9862a1c5e